Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/DownloadObject.cs

```diff
@@ -144,7 +129,7 @@ private static bool FileLocked(int fileLockedRetries, string fullPath, Cancellat
     {
```

Comment:
https://github.com/FrendsPlatform/Frends.AmazonS3/blob/a9100447af685bd8b50b90b4e33438f6915fba22/Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/DownloadObject.cs#L128
You're not using this method, so it can be removed.

Reply:
Good that you noticed, blind trust to VS is not a good idea. 
This should've been used when overwriting file and it's fixed now.